src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -265,7 +262,7 @@ void manualRegistrationToolBox::stopManualRegistration()
 
 void manualRegistrationToolBox::synchroniseMovingFuseView()
 {
-    if(this->parentToolBox()) //no need in Reg. workspace
+    if(selectorToolBox()) //no need in Reg. workspace
```

Comment:
according to the comment, this line is not needed, so either we keep this condition and remove the comment, or remove the whole condition. In fact, i do not think either that we need it, the selector has to be valid everytime.